addons/mail/models/mail_mail.py

```diff
@@ -541,7 +541,10 @@ def send(self, auto_commit=False, raise_exception=False):
                     len(batch_ids), mail_server_id)
             finally:
                 if smtp_session:
-                    smtp_session.quit()
+                    try:
+                        smtp_session.quit()
+                    except Exception:
```

Comment:
Why don't you respect the "raise_exception" flag ? If it is wanted (could makes sense since we sent emails), if is worth a comment.